Pod/Classes/MenuView.swift

```diff
@@ -8,6 +8,16 @@
 
 import UIKit
 
+public struct MultilineMenuItem: MenuItemType {
```

Comment:
Not even for this?
```
PagingMenuController(menuItemTypes: ["Viewable Menu1", "Viewable Menu 2"], options: options)
```

if we make this internal, MultilineMenuItem would not work for menuItemTypes.